app/views/assignments/_boot.js.erb

```diff
@@ -45,7 +45,7 @@
     $('.assignment_file').each(function() {
       if (!filename_re.exec(this.value)) {
         invalid = true;
-        // _constants.scss $red
+        // _constants.scss $severe-error
         $(this).css('background-color', '#fe2a2a');
```

Comment:
I think you can set this to a `--var(...)`? In which case you can also remove the comment.

Reply:
Ok, I made the change but I'm not too sure what `check_required_fields()` does

Reply:
It's doing some client-side validation for the assignments form (enabling/disabling the "Submit" button)